### PR TITLE
dx-1290 corrected to paymentorder and fixed screenshot

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

**Review by arebra — APPROVED** [review]
Looks great to me 👍

File: _includes/transaction-on-file.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+
+{% assign api_resource=include.api_resource %}
+{% capture url_token %}
+    /psp/{{ api_resource }}
+    {%- if api_resource == "creditcard" -%}
+        /payments
+    {%- endif -%}
+{% endcapture %}
+{% assign url_token = url_token | strip %}
+{% assign url_without_psp = url_token | remove: "/psp/" %}
```
Comment (asbjornu) [inline _includes/transaction-on-file.md:10]: 1. Turning this on its head makes it more robust by not having to `remove` the `/psp/` prefix.
2. I think we should change the variable names, so please adjust the rest of the include accordingly.
3. I think it's more future-proof to check whether `api_resource` is different from `paymentorders` than equal to `creditcard`.

```suggestion
{% assign token_url_without_psp = include.api_resource %}

{% if include.api_resource != "paymentorders" %}
    {% assign token_url_without_psp = token_url_without_psp | append: '/payments' %}
{% endif %}

{% assign token_url = token_url_without_psp | prepend: '/psp/' %}
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/transaction-on-file.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+
```
Comment (asbjornu) [inline _includes/transaction-on-file.md:1]:
```suggestion
```

**Review by asbjornu — APPROVED** [review]

**Review by asbjornu — APPROVED** [review]
🎉

**Pull request merged**